src/components/Filterbar/FilterBar.jsx

```diff
@@ -1,51 +1,59 @@
-import React from 'react'
+import React from "react";
 import { IoReload } from "react-icons/io5";
 import { FaRegStar } from "react-icons/fa";
 import "../Filterbar/FilterBar.css";
-import { useProduct } from '../../context/product-context';
+import { useProduct } from "../../context/product-context";
 
+export const FilterBar = () => {
+  const { state, dispatch } = useProduct();
+  const {
+    sortByPrice,
+    pricerange,
+    showAllProduct,
+    fastDelivery,
+    sortByQuantity,
+    sortByRatings,
+    sortByBrand,
+  } = state;
 
- export const FilterBar = () => {
-   const {state,dispatch}=useProduct();
-   const {sortByPrice,pricerange,showAllProduct,fastDelivery,sortByQuantity,sortByRatings,sortByBrand}=state;
-
-   function BrandClickHandler (event) {
-    let checkedBrand =event.target.checked;
-    const  brandName = event.target.value;    
-    if(checkedBrand)
-    {
-        return {type:"SET_BRAND_NAME", payload:brandName}
-    }
-    else{
-        return {type:"REMOVE_BRAND_NAME", payload:brandName}
+  function BrandClickHandler(event) {
+    let checkedBrand = event.target.checked;
+    const brandName = event.target.value;
+    if (checkedBrand) {
+      return { type: "SET_BRAND_NAME", payload: brandName };
+    } else {
+      return { type: "REMOVE_BRAND_NAME", payload: brandName };
```

Comment:
You can return it directly without a `else` block

Reply:
done